articles/advisor/advisor-reference-reliability-recommendations.md

```diff
@@ -645,6 +645,23 @@ In active-active configuration, both instances of the VPN gateway establish S2S
 
 Learn more about [Virtual network gateway - VNetGatewayActiveActive (Enable Active-Active gateways for redundancy)](https://aka.ms/aa_vpnha_learnmore).
 
+### Use HEAD health probes
+
+Front Door Health probes can use either the GET or HEAD HTTP method. It’s a good practice to use the HEAD method for health probes, which reduces the amount of traffic load on your origins.
+
```

Comment:
For health probes, it’s a good practice to use the HEAD method, which reduces the amount of traffic load on your origins.